pkg/mover/doc_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 )
 
-// Package level documentation
+// Move a chart and its dependencies to another registry and or repository
```

Comment:
can't they be in the same file? Do we need 3 files for 3 examples?

Reply:
seems so:

https://go.dev/blog/examples
> Multiple examples can be provided for a given identifier by using a suffix beginning with an underscore followed by a lowercase letter. Each of these examples documents the Reverse function:
> 
> func ExampleReverse()
> func ExampleReverse_second()
> func ExampleReverse_third()

A single long code example would be confusing IMHO.

Reply:
Maybe I can put all in the same file and still have 3 separate samples.

Reply:
yep, that seems to work fine. Will be fixed in next update.

Reply:
done